### DISCUSSION TITLE
Wanted: Plugin bisect (with exclusion)

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#7588] [Ideas]
https://code.visualstudio.com/blogs/2021/02/16/extension-bisect help me dicover bad extension two times https://github.com/gitbito/bitoai/issues/13

And today one guy in QQ group say https://github.com/Gk0Wk/TW5-CodeMirror-Enhanced not working in https://keatonlao.github.io/tiddlywiki-xp/ template, and don't know why.

I think this can be addressed by a bisect plugin, that can set itself+core+language+TW5-CodeMirror-Enhanced to be excluded from bisect, and do bisect on the rest of the plugins, to find out the conflicted plugin with interested plugin (TW5-CodeMirror-Enhanced in this case).

**Comment by Jermolene** [comment]
Hi @linonetwo if I'm understanding correctly, you're suggesting a new "bisect" plugin that guides the user through diagnosing problems with plugins by progressively enabling/disabling them according to a bisect algorithm. That seems like a very interesting idea.

**Reply by linonetwo** [reply]
Yes, just a todo about a new plugin. We already have necessary APIs, just need some time.
